2020-05-23 14:20:04,299 [main] ERROR [com.mimosa.deeppokemon.crawler.LadderBattleCrawler] LadderBattleCrawler.java:154 - d != java.time.LocalDate
2020-05-23 15:08:30,645 [main] ERROR [com.mimosa.deeppokemon.crawler.RegexTeamCrawler] RegexTeamCrawler.java:39 - Software caused connection abort: recv failed
2020-05-23 15:10:21,666 [main] ERROR [com.mimosa.deeppokemon.crawler.RegexTeamCrawler] RegexTeamCrawler.java:39 - Read timed out
2020-05-23 15:10:53,472 [main] ERROR [com.mimosa.deeppokemon.crawler.RegexTeamCrawler] RegexTeamCrawler.java:39 - Read timed out
2020-05-23 15:44:59,499 [main] ERROR [com.mimosa.deeppokemon.service.BattleService] BattleService.java:27 - E11000 duplicate key error collection: deep_pokemon_test.battle index: _id_ dup key: { _id: "gen8ou-1119226837" }; nested exception is com.mongodb.MongoWriteException: E11000 duplicate key error collection: deep_pokemon_test.battle index: _id_ dup key: { _id: "gen8ou-1119226837" }
2020-05-23 15:45:49,013 [main] ERROR [com.mimosa.deeppokemon.service.BattleService] BattleService.java:27 - E11000 duplicate key error collection: deep_pokemon_test.battle index: _id_ dup key: { _id: "gen8ou-1119226837" }; nested exception is com.mongodb.MongoWriteException: E11000 duplicate key error collection: deep_pokemon_test.battle index: _id_ dup key: { _id: "gen8ou-1119226837" }
2020-05-23 15:49:41,114 [main] ERROR [com.mimosa.deeppokemon.service.BattleService] BattleService.java:28 - E11000 duplicate key error collection: deep_pokemon_test.battle index: _id_ dup key: { _id: "gen8ou-1119226837" }; nested exception is com.mongodb.MongoWriteException: E11000 duplicate key error collection: deep_pokemon_test.battle index: _id_ dup key: { _id: "gen8ou-1119226837" }
